src/esperanza/walletextension.cpp

```diff
@@ -13,6 +13,7 @@
 #include <esperanza/finalizationstate.h>
 #include <esperanza/script.h>
 #include <finalization/state_repository.h>
+#include <injector.h>
```

Comment:
What is this include needed for?

It should not be necessary to include `injector.h` anywhere but in `init.cpp` or in legacy bitcoin code which is not a component. The wallet extension I have to admit is Frankensteins Component, but does have the walletextension_deps to cope with components and dependencies.

Anyways I can not see any particular usage of anything exported by this include.

Reply:
That was a leftover. I began with `GetComponent` and then moved the dependency to `WalletExtensionDeps`.